src/main/java/core/basesyntax/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String INPUT_DATA = "fruit_shop_transactions.csv";
+
+    @Override
+    public List<FruitTransaction> readFile(String fileName) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(fileName))) {
```

Comment:
It's recommended to use try-with-resources to ensure that the BufferedReader is closed properly. Good job using it here.